src/js/main.js

```diff
@@ -248,6 +248,10 @@ function startProcess() {
             const tab = tabClass.substring(4);
             const tabName = $(self).text();
 
+            if (GUI.active_tab === 'pid_tuning') {
```

Comment:
This is probably an edge case, but we should ensure here that the restore can not be executed if the user clicks 'Save & Reboot' in the PID Tuning tab.

Reply:
When clicking on another tab while in pid_tuning and something has changed we have to restore to initial settings to avoid user confusion thinking there was nothing saved.

Reply:
Added condition for restore.